bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -452,20 +457,16 @@ contract NFTBridge is
     );
   }
 
-  function setFeePercentage(uint256 amount) external onlyOwner {
-    require(
-        amount < (FEE_PERCENTAGE_DIVIDER / 10),
-        "Bridge: bigger than 10%"
-    );
-    feePercentage = amount;
-    emit FeePercentageChanged(feePercentage);
+  function setFixedFee(uint256 amount) external onlyOwner {
```

Comment:
shouldn't the fee be always greater than 0?

Reply:
We may want to set this to 0 and subvention the bridge to get people using it

Reply:
@pmprete I remember about an Idea to let the possibility to set as 0, but not the reason for that

Reply:
We may want to make the service free to attract people, once the bridge is more popular we will add a fee

Reply:
sounds reasonable!